### PR TITLE
Created a separate class for all phidget bridge related functions, wi…

### PR DESCRIPTION
**Pull request by jchung55**
Please review these two files: phidget_bridge, test_phidget_bridge

---
This change is [Reviewable](https://reviewable.io/reviews/jhu-asco/esc_closed_loop_force_control/1)

### CODE REVIEW

File: phidget_bridge.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+from Phidget22.Devices.VoltageRatioInput import VoltageRatioInput
+from Phidget22.PhidgetException import PhidgetException
+from Phidget22.Phidget import ChannelSubclass
+
+class PhidgetBridge:
```
Comment (garimellagowtham) [inline phidget_bridge.py]: Add description about the class.
Something like wrapper class to read in force from Phidgets voltage bridge

File: phidget_bridge.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    # Create constructor  
+    def __init__(self, frequency = 20.0):
+        self.connected_status = False
```
Comment (garimellagowtham) [inline phidget_bridge.py]: Add comments about member variables

File: test_phidget_bridge.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+from phidget_bridge import PhidgetBridge
+
+i = 0
+# Specifies the frequency at which the PhidgetBridge will take in data
+frequency = 10
+time_off = 0.1
+# Calls class with a specified frequency
+phidget_bridge = PhidgetBridge(frequency)
+# Calls function to wait for connection
+phidget_bridge.waitingForConnection(timeout = 5000)
```
Comment (garimellagowtham) [inline test_phidget_bridge.py]: After this line check phidget is connected. If not exit

File: test_phidget_bridge.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import time
+from phidget_bridge import PhidgetBridge
+
+i = 0
+# Specifies the frequency at which the PhidgetBridge will take in data
+frequency = 10
+time_off = 0.1
```
Comment (garimellagowtham) [inline test_phidget_bridge.py]: No comment here?
"Time interval between force data collection"

File: test_phidget_bridge.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import time
```
Comment (garimellagowtham) [inline test_phidget_bridge.py:6]: Add a few comment lines on what this file does. Something like:
" Testing Phidget Bridge! Instantiate and collect force data from the load cell.

  Author: Joseph Chung
  Email:
"

**Review by garimellagowtham — CHANGES_REQUESTED** [review]
Looking Good!

**Comment by msheckells** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 7 unresolved discussions.

---

*[phidget_bridge.py, line 24 at r1](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1#-KomhmZeuzzCrTFGrO-N:-KomhmZeuzzCrTFGrO-O:b-rypyev) ([raw file](https://github.com/jhu-asco/esc_closed_loop_force_control/blob/99dcbeac8754025de836ada597e0b923506df5cd/phidget_bridge.py#L24)):*
> ```Python
>         self.ch.setOnVoltageRatioChangeHandler(self.voltageRatioChangeHandler)
>     
>     # Function to wait for a connection to the PhidgetBridge    
> ```

In general, function documentation should describe the function parameters as well, e.g.

timeout Time to wait for connection in milliseconds

---

*[test_phidget_bridge.py, line 13 at r1](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1#-Komj1VMTorEsRfFykmu:-Komj1VMTorEsRfFykmv:b-at231w) ([raw file](https://github.com/jhu-asco/esc_closed_loop_force_control/blob/99dcbeac8754025de836ada597e0b923506df5cd/test_phidget_bridge.py#L13)):*
> ```Python
> phidget_bridge.waitingForConnection(timeout = 5000)
> # While iterating 10 times, prints the force and sleeps for a specified time
> while i < 10:
> ```

Use `for i in range(10):` instead of a while loop with  a counter.  It is cleaner in this case.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Review by garimellagowtham — APPROVED** [review]

**Comment by garimellagowtham** [comment]
Reviewed 1 of 2 files at r2.
Review status: 1 of 2 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[phidget_bridge.py, line 97 at r2](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1#-Kosz8uadWHyAXNYl0F4:-Kosz8uadWHyAXNYl0F5:b62nn0k) ([raw file](https://github.com/jhu-asco/esc_closed_loop_force_control/blob/2a90546757f939b6dc927f16d371d97ff220848f/phidget_bridge.py#L97)):*
> ```Python
>     def close(self):
>         self.ch.close()
>         
> ```

Remove spaces

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Comment by garimellagowtham** [comment]
Review status: 0 of 3 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[test_phidget_bridge.py, line 30 at r3](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1#-KotofJCcMvwyLDb9M1D:-KotofJCcMvwyLDb9M1E:b3f5t1q) ([raw file](https://github.com/jhu-asco/esc_closed_loop_force_control/blob/1504db7d2ac21c4bb1e11499082ca99212de0467/test_phidget_bridge.py#L30)):*
> ```Python
>     print(phidget_bridge.getForce())
>     time.sleep(time_off)
>     i += 1
> ```

You do not need to increase i in this for loop.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Comment by msheckells** [comment]
Reviewed 3 of 3 files at r3.
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Comment by garimellagowtham** [comment]
Reviewed 1 of 2 files at r4.
Review status: 2 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Comment by msheckells** [comment]
Reviewed 1 of 2 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/jhu-asco/esc_closed_loop_force_control/1)*

**Pull request merged**